RadixWallet/Features/SettingsFeature/Settings+Reducer.swift

```diff
@@ -175,4 +183,13 @@ extension Settings {
 			)))
 		}
 	}
+
+	private func loadSecurityProblems() -> Effect<Action> {
```

Comment:
I'd call this securityProblemsEffect

Reply:
I think it is more descriptive with the `load` prefix, plus seems to be the standard followed
![Screenshot 2024-05-15 at 15 44 22](https://github.com/radixdlt/babylon-wallet-ios/assets/164921079/f4c1025f-f25a-44fe-8464-ae5b3aeaec17)

Reply:
It isn't loading anything though, and we already have exactly these mechanics in many other places, so we should use the same naming conventions. Please model it on SecurityCenter+Reducer, where I do exactly the same thing.

Reply:
Updated both methods from this class, but to me the `securityProblemsEffect` isn't descriptive enough. We could call it `read` instead of `load` if you want, but the name of the method should describe what's actually happening.

Reply:
The convention is that for streams we just use the plural of the thing that the stream produces. This is based on the fact that apple explicitly discourages using things like "get..." as prefix for async methods.

Reply:
So load or read would make sense for something you fetch once, or at least something you fetch using "pull" logic. For something that is "push", like a stream, load does not make much sense in my mind, and in any case we started using the convention above so we better stick to it.